content/60.methods.md

```diff
@@ -211,7 +211,7 @@ This figure is included in an analysis of pharmaceutical development efforts dur
 
 The seven manuscripts produced by the consortium (outside of this one) will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews so that they can continue to be updated as more information becomes available.
 This approach allows for a version of record to be maintained alongside the most recent developments, which are always available through GitHub.
-Currently, three manuscripts have been submitted, with one accepted for publication [@individual-nutraceuticals] and two under revision [@individual-pathogenesis; @individual-pharmaceuticals].
+The first of these manuscripts was recently published [@doi:10.1128/mSystems.00122-21].
```

Comment:
Wow!  I didn't realize it was out!  We should keep the `@individual-nutraceuticals` citation here but change the reference in `content/90.back-matter.md` to use this DOI instead of the arXiv citaiton.

In the lines above, how confident are we that we will still finish all seven manuscripts?  Do we need to hedge a little more in the phrasing in case some of them remain Manubot-only and aren't submitted to the journal?

Reply:
I think Casey is still planning that we will submit all 7 -- I should hopefully have more bandwidth starting next week (no longer moonlighting at Haverford) to push them across the finish line!

Reply:
Awesome!  Then let's leave it.